app/components/filters_component/filters_component.scss

```diff
@@ -12,8 +12,6 @@
     position: relative;
 
     &-link-button {
```

Comment:
I think `&-link-button` can be completely removed as this is a link, not a button link. If you completely remove it, it will look like a link, which is less misleading I think